mercadopago/resources/order.py

```diff
@@ -33,3 +32,131 @@ def create(self, order_object, request_options=None):
             raise ValueError("Param order_object must be a Dictionary")
 
         return self._post(uri="/v1/orders", data=order_object, request_options=request_options)
+
+    def get(self, order_id, request_options=None):
+        """[Click here for more info](https://www.mercadopago.com.br/developers/en/reference/order/online-payments/get-order/get ) # pylint: disable=line-too-long
+
+        Args:
+            order_id (str): The Order ID
+            request_options (mercadopago.config.request_options, optional): An instance of
+            RequestOptions can be pass changing or adding custom options to ur REST call.
+            Defaults to None.
+
+        Raises:
+        ValueError: Param order_id must be a string
+
+        Returns:
+            dict: Order ID returned in the response to the request made for its creation.
```

Comment:
[req] O método retorna a resposta inteira da API de Order e não somente o ID
```suggestion
            dict: Order returned in the response to the request made for its creation.
```